server/routes/reporting/standard-report/standard-report-print.js

```diff
@@ -368,6 +372,10 @@ function resolveSortBy(sortBy, reportData) {
   return reportData.defaultSortColumn;
 }
 
+function isNumber(n) {
+  return !isNaN(parseFloat(n));
+}
```

Comment:
Need to update this as currently this will return true for dates, i.e '2024-01-01'
Which messes up sorting for those dates.

Maybe something along the lines of:
`return !isNaN(parseFloat(n)) && !(moment(n, 'yyyy-MM-DD', true).isValid());`

Not sure if all dates come in that format, but i think they do

Reply:
make sure moment is imported or it will crash :)

Reply:
my bad 🤓 updated